test/test-legacy.js

```diff
@@ -1,4 +1,4 @@
-/* globals describe, it */
+/* globals before, describe, it */
```

Comment:
btw `/* eslint-env mocha */` might be a good shortcut to remember to file away in your brain

Reply:
i like reminding myself how many silly globals this test framework is making me use.